services/invitations/src/simcore_service_invitations/api/_invitations.py

```diff
@@ -42,16 +41,15 @@ async def create_invitation(
 ):
     """Generates a new invitation code and returns its content and an invitation link"""
 
-    invitation_link = create_invitation_link(
+    invitation_link, invitation_content = create_invitation_link_and_content(
         invitation_inputs,
         secret_key=settings.INVITATIONS_SECRET_KEY.get_secret_value().encode(),
         base_url=settings.INVITATIONS_OSPARC_URL,
         default_product=settings.INVITATIONS_DEFAULT_PRODUCT,
```

Comment:
One more question, are you sure we want to have a default product? I would be more defensive and when the product is not specifically provided I would fail.

Reply:
I would leave this decision on the client side (he can implement default on his side if it is wanted)

Reply:
yes, the default product is necessary for backwards compatibility. Current systems running against a previous version of this service do not define a product in the request. But now I always have to include one (together with the emai).

Wait, this not the default product from the database (NOTE that this system has no database!)
Perhaps I should rename it differently

Reply:
ok understood, no its fine then, thanks